message_definitions/v1.0/ASLUAV.xml

```diff
@@ -140,13 +140,15 @@
     </message>
     <message id="209" name="SENS_BATMON">
       <description>Battery pack monitoring data for Li-Ion batteries</description>
+      <field type="uint64_t" name="batmon_timestamp"> Battery monitor last timestamp </field>
       <field type="float" name="temperature">Battery pack temperature in [deg C]</field>
```

Comment:
I guess int16_t, and the values in centi degrees C (1/100 deg C) would also work just fine, while saving 2 bytes.
But this can be done another time maybe...